docs/standard/native-interop/interop-exceptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Platform Neutral Interop Exceptions"
```

Comment:
```suggestion
title: "Exceptions interoperability"
```

Reply:
I believe "Interop Exceptions" would mean "exceptions that are meant to be used for interop". It is not what this doc is about.

Reply:
I had misunderstood the concept this whole time. Thanks for the clarification!

Reply:
Maybe we should also rename the file to match the title? To `exceptions-interoperability.md`?

Reply:
Looking at the files under https://github.com/dotnet/docs/tree/master/docs/standard/native-interop , there does not seem to be uniform file naming scheme.

I will leave it up to you to pick the filename. I do not have strong opinion.